pyaedt/edb_core/ipc2581/ecad/cad_data/primitives/path.py

```diff
@@ -0,0 +1,85 @@
+import xml.etree.cElementTree as ET
+
+from pyaedt.edb_core.ipc2581.content.entry_line import EntryLine
+from pyaedt.edb_core.ipc2581.ecad.cad_data.primitives.polygon import PolyStep
+from pyaedt.edb_core.ipc2581.ecad.cad_data.primitives.polygon import PolyType
+
+
+class Path(object):
+    """Class describing ipc2581 traces."""
```

Comment:
```suggestion
    """Class describing an IPC2581 trace."""
```
Shouldn't this be singular "trace"? The others are singular.